addons/website/static/src/components/dialog/dialog.js

```diff
@@ -0,0 +1,51 @@
+/** @odoo-module **/
+
+import { Dialog } from '@web/core/dialog/dialog';
+import { _lt } from 'web.core';
+
+const NO_OP = () => {};
+
+export class WebsiteDialog extends Dialog {
```

Comment:
> WIP - not sure that component is 100% useful.

Mmh yes, I imagine we should be happy enough with standard backend dialog & action behavior right? Just to be consistent with other apps?